erpnext/public/js/controllers/transaction.js

```diff
@@ -498,7 +498,35 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 
 	item_code(doc, cdt, cdn) {
 		var me = this;
+		// Experimental: This will be removed once stability is achieved.
+		frappe.db.get_single_value('Selling Settings', 'use_server_side_reactivity')
```

Comment:
This will make a new request every time... better to pass it with the boot. [example](https://github.com/frappe/erpnext/blob/develop/erpnext/startup/boot.py)

Reply:
Makes sense.